<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurfaceContouring</class>
 <widget class="QDialog" name="SurfaceContouring">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>曲面自动重建向导</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="OsgContainer" name="osgContainer" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="controlPanel" native="true">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>初始数据</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="openSampleFile">
           <property name="text">
            <string>打开仿形数据</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="filePath">
           <property name="text">
            <string>仿形数据地址</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="openCloudPointsFile">
           <property name="text">
            <string>打开点云数据</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="filePathCloudPoint">
           <property name="text">
            <string>点云数据地址</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>手动采集仿形数据</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelSampleDensity">
           <property name="text">
            <string>每行采点个数</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="numInLine">
           <property name="minimum">
            <number>3</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelSample">
           <property name="text">
            <string>待采点编号</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelSampleIndex">
           <property name="text">
            <string>第1行，第1个点</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="sample">
           <property name="text">
            <string>手动采点</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="finishConstruction">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>完成仿形</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioFeed">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>步进模式</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioCloud">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>点云模式</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>3</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="feedMode">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>自动单向步进模式</string>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="2" column="0">
          <widget class="QPushButton" name="startFeed">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>开始步进</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="resetFeed">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>重新步进</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>扫查速度(&lt;10w)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>水距(mm)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="waterPath">
           <property name="minimum">
            <double>2.000000000000000</double>
           </property>
           <property name="maximum">
            <double>200.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>5.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="speed">
           <property name="maximum">
            <number>100000</number>
           </property>
           <property name="singleStep">
            <number>1000</number>
           </property>
           <property name="value">
            <number>20000</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="cloudMode">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>点云采集模式</string>
        </property>
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="firstCloudMode">
           <property name="text">
            <string>基于采样点</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="startCloudMode">
           <property name="text">
            <string>基于点云</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OsgContainer</class>
   <extends>QWidget</extends>
   <header>OsgContainer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
